snapcraft.yaml

```diff
@@ -1045,7 +1045,7 @@ parts:
   virtiofsd:
     source: https://gitlab.com/virtio-fs/virtiofsd
     source-depth: 1
-    source-tag: v1.8.0
+    source-tag: v1.10.1
```

Comment:
According to changes on https://gitlab.com/virtio-fs/virtiofsd/-/tags, they are fixing stuff for ppc64el, another reason for me to test this bit on non-amd64 :)